cmake/Modules/FindLIBSAMPLERATE.cmake

```diff
@@ -0,0 +1,11 @@
+if(NOT LIBSAMPLERATE_LIBRARY)
+  if(LIBSAMPLERATE_LIBRARY_DIRECTORY)
+    find_library(LIBSAMPLERATE_LIBRARY NAMES samplerate libsamplerate-0 HINTS "${LIBSAMPLERATE_LIBRARY_DIRECTORY}")
+  else()
+    find_library(LIBSAMPLERATE_LIBRARY NAMES samplerate libsamplerate-0)
+  endif()
```

Comment:
I think you can just add this to hints whether it's empty or not and simplify this.

Reply:
nice I didn't know, I will fix!

Reply:
fixed